src/components/calcite-pick-list-item/readme.md

```diff
@@ -1,5 +1,7 @@
 # calcite-pick-list-item
 
+The `calcite-pick-list-item` is a child element of `calcite-pick-list`. Items are displayed and can be selected or deselected to be shown on the view.
```

Comment:
Let's describe the component a bit more. It will show either a radio button or checkbox, and accepts a secondaryAction slot for custom behavior per item.